README.md

```diff
@@ -6,7 +6,7 @@ Recent Computer Science graduate with a background in Behavior Analysis; I have
 
 **Proficient:** OOP, Java, SpringBoot, Maven, Linux/Bash, Docker, Git/GitHub, RDBMS/SQL, Agile/Scrum
 
-**Competent:** Big Data (Hadoop), Notebooks (Jupyter, Zeppelin), Python (NumPy, Pandas), REST APIs, Eiffel, C, Go, HTML/CSS/JavaScript, Android Development
+**Competent:** Big Data (Spark, Hadoop), Notebooks (Zeppelin, Databricks, Jupyter), Python (PySpark, NumPy, Pandas), REST APIs, Eiffel, C, Go, HTML/CSS/JavaScript, Android Development
```

Comment:
~Big Data (Spark, Hadoop)~  Spark, Hadoop  (people know they are big data tools), 
~Notebooks (~ Zeppelin, Databricks, Jupyter ~)~
Python, Numpy, Pandas

Move Python to the beginning (order matters)